### ISSUE TITLE
species stats: synonym count

### ISSUE THREAD

**Issue by bpescador** [calacademy-research/antcat#275]
In the example below, what does "(2 synonyms)" refer to? If it is the count of synonyms, there are 7 junior synonyms.  

http://antcat.org/catalog/457338
Lioponera longitarsus Mayr, 1879 valid This taxon has been changed; changes awaiting approval
(2 synonyms)

More statistics
Lioponera longitarsus Mayr, 1879: 667 (w.q.) INDIA. Indomalaya. AntWeb AntWiki HOL
Forel, 1900f: 329 (m.).
Combination in Cerapachys: Brown, 1975: 23.
Combination in Lioponera: Borowiec, 2016: 164.
Status as species: Bingham, 1903: 27; Wheeler, 1918a: 264.
Senior synonym of Lioponera longitarsus australis: Forel, 1895f: 422.
Senior synonym of Lioponera bicolor, Lioponera longitarsus parva, Lioponera pygmaeus: Brown, 1975: 23.
Senior synonym of Lioponera aegyptiacus, Lioponera alfierii, Lioponera cooperi: Bolton, 1995b: 143. Changed by Brendon E. Boudinot about 1 year ago

**Comment by jonkerz** [comment]
> In the example below, what does "(2 synonyms)" refer to?

It's the statistics for children of *Lioponera longitarsus*, so in this case it refers to 2 subspecies that are synonyms. The stats would make more sense if *L. longitarsus* had valid subspecies, because then it would say "5 valid subspcies (2 synonyms)". This was changed in #278 so that it it says "0 valid subspecies (2 synonyms)" when there's no valid subspecies.

>  If it is the count of synonyms, there are 7 junior synonyms.

As mentioned above, it's not, but we want this feature so I created an issue for it, see #279.
